src/main/java/games/strategy/engine/framework/GameDataUtils.java

```diff
@@ -33,6 +38,36 @@ public static GameData cloneGameData(final GameData data, final boolean copyDele
     }
   }
 
+  /**
+   * Clones a single GameData object multiple times.
+   * This is faster than calling {@link GameDataUtils#cloneGameData(GameData)} multiple times,
+   * because the original GameData object is only getting serialized once.
+   * 
+   * @param data The {@link GameData} to be cloned.
+   * @param times The targeted amount of {@link GameData} objects the returned {@link List} should contain.
+   * @return An unmodifiable {@link List} that containing clones of the passed data
+   */
+  public static List<GameData> cloneGameData(final GameData data, final int times) {
+    try {
+      final byte[] bytes = IoUtils.writeToMemory(os -> GameDataManager.saveGame(os, data, false));
```

Comment:
For some reason, I thought the `cloneGameData()` methods acquired a read lock on the `GameData`.  The comment in the overload above indicates that is not the case.  So, I would suggest, for this overload, we acquire the read lock during serialization only.

We should probably document that, as well.  Not sure if Javadocs, as above, are more appropriate, or applying `@ThreadSafe` to the method is the way to go here.